src/index.js

```diff
@@ -28,12 +29,17 @@ const location = `${thisDocument.location.protocol}//${thisDocument.location.hos
 const socket = io.connect(location);
 
 const createStoreWithMiddleware = applyMiddleware(
+  authenticationMiddleware(socket),
   remoteActionMiddleware(socket),
   loggerMiddleware({ collapsed: true })
 )(createStore);
 const store = createStoreWithMiddleware(reducer, window.__INITIAL_STATE__);
 
 socket.on('connected', (data) => console.log(data));
+socket.on('authenticated', ({ user, token }) => {
+  document.cookie = `_posToken=${token}`;
+  store.dispatch(setUser({ user, token }));
+});
 socket.on('state', (state) => store.dispatch(setState(state)));
```

Comment:
@achiu8 this is what i was referring to. After refreshing the page, the client, for some reason, stops receiving this event. The server is still emitting it, though...